sample_code/jwt-sample-app/index.js

```diff
@@ -33,32 +33,25 @@ const S3 = new AWS.S3({ apiVersion: '2006-03-01', signatureVersion: 'v4' })
 const statusRetryMillis = 200
 const statusRetries = 20
 
-writeStatus('1. STARTING')
-
 /* Given the settings in the config file, create a JWT */
-function generateJWT (cb) {
-  writeStatus('2. Generate JWT')
+async function generateJWT () {
```

Comment:
Consider replacing the generateJWT and requestServiceToken methods with the [@adobe/jwt-auth](https://www.npmjs.com/package/@adobe/jwt-auth) package.

```
const auth = require('@adobe/jwt-auth');
 
let token = await auth(config);
```

If you'd like I can send a PR on this.

Reply:
Cool, I'll make this change.

Reply:
Let me know if you run into any issues. I wrote the package to make life for internal and external devs easier.

Reply:
Hmm, yes. I'll contact you offline.

Reply:
Works now.